learning/dashboard/webserver/readme.md

```diff
@@ -0,0 +1,161 @@
+
+**Microsoft IIS :**
+
+A proprietary web server created by Microsoft for use with the Windows operating system is called IIS (Internet Information Services). With support for many different web technologies and protocols, it is a strong and feature-rich web server. Enterprise environments frequently employ IIS, which is connected with other Microsoft goods like Active Directory and Exchange.
+Static web pages, dynamic web pages, and web-based apps can all be hosted on IIS, a powerful and expandable web server. It may be customised with a range of modules and extensions to add new features and capabilities. It supports a variety of programming languages, including ASP.NET, PHP, and Python.
+
+**Lighttpd :**
+
+A free and open-source web server called Lighttpd was created to be small, quick, and effective. Because it is built in C and has a small memory footprint, it is a viable option for hosting busy websites on servers with limited resources.
+One of Lighttpd's primary characteristics is its modular architecture, which enables users to quickly alter and expand the server's capabilities. FastCGI, URL rewriting and server-side programming languages like PHP and Ruby are just a few of the features it provides. Moreover, it can be set up to operate with external authentication modules and includes built-in support for SSL/TLS.
+
+**Jigsaw :**
+
+The World Wide Web Consortium (W3C) created the web server Jigsaw as a reference implementation of the HTTP/1.1 protocol. For programmers curious about HTTP protocols and web server technology, Jigsaw is still a popular option.
+Jigsaw's modular design, which enables customers to easily adapt and increase the server's capability, is one of its primary benefits. CGI, server-side programming languages like PHP and Python, and SSL/TLS encryption are just a few of the features it provides. Moreover, it includes built-in support for virtual hosting, which enables the hosting of numerous websites on a single server.
+
+#	How to setup and configure Nginx ?
+
+##	Nginx installation on Ubuntu 20.04
+
+A package manager is typically included with every operating system. You have a choice between using apt or apt-get with Ubuntu 20.04. As this is a new server, you first need to update the package manager.
+
+Step 1 :  Upgrade the package management to the most recent version that is compatible:
+You can either use the apt or apt-get package manager to install Nginx.
+'''sudo apt updatesudo apt-get update'''
+Now, you are ready to install Ngnix. When you are logged in with a non-root account with sudo rights, you will need to prefix every task that requires admin privileges with sudo.
+
+Step 2 : Install Ngnix web server using apt:	
+'''sudo apt install nginxsudo apt-get install nginx'''
```

Comment:
This doesn't look right. Use single backticks for single line code. Use triple backticks for code blocks.